src/services/user.service.js

```diff
@@ -1,6 +1,9 @@
 const httpStatus = require('http-status');
 const { User } = require('../models');
 const ApiError = require('../utils/ApiError');
+const { OAuth2Client } = require('google-auth-library');
+const client = new OAuth2Client(process.env.GOOGLE_CLIENT_ID);
```

Comment:
Could you move `process.env.GOOGLE_CLIENT_ID` into the config and bring that in so it's parsed and known if exists?